src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -139,108 +149,134 @@ public OpenIdConnectHandler(HttpClient backchannel, HtmlEncoder htmlEncoder)
         }
 
         /// <summary>
-        /// Handles Signout
+        /// Sign out the Relying Party from the OpenID provider 
         /// </summary>
-        /// <returns></returns>
-        protected override async Task HandleSignOutAsync(SignOutContext signout)
+        /// <returns>A task executing the sign out procedure</returns>
+        protected override async Task HandleSignOutAsync(SignOutContext context)
         {
-            if (signout != null)
+            if (context == null)
```

Comment:
Except that would have null reffed right here:
https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNetCore.Authentication/AuthenticationHandler.cs#L269